pages/apply/[resident]/personal-details.tsx

```diff
@@ -71,25 +110,182 @@ const ApplicationStep = (): JSX.Element => {
         markAsComplete: true,
       })
     );
-  };
 
-  const onSubmit = async () => {
     router.push(`/apply/${resident}`);
   };
 
+  const options = [
+    {
+      label: 'Select an option',
+      value: '',
+    },
+    {
+      label: 'Mr',
+      value: 'Mr',
+    },
+    {
+      label: 'Mrs',
+      value: 'Mrs',
+    },
+    {
+      label: 'Miss',
+      value: 'Miss',
+    },
+    {
+      label: 'Ms',
+      value: 'Ms',
+    },
+    {
+      label: 'Mx',
+      value: 'Mx',
+    },
+    {
+      label: 'Other',
+      value: 'Other',
+    },
+  ];
+
+  const radioProps: RadioConditionalProps = {
+    value: '',
+    as: 'radioconditional',
+    label: 'Gender',
+    name: 'gender',
+    options: [
+      {
+        label: 'Male',
+        value: 'M',
+      },
+      {
+        label: 'Female',
+        value: 'F',
+      },
+      {
+        label: 'Prefer to self-describe',
+        value: 'self',
+        conditionalFieldInput: {
+          as: 'input',
+          containerId: 'self-describe-text-values',
+          fieldId: 'self-describe',
+          fieldName: 'self-describe',
+          label: 'Please enter your self-description',
+          display: true,
+        },
+      },
+    ],
+  };
+
+  function generateValidationSchema(state: State, applicant: Applicant) {
+    const min = Math.min(+new Date());
+
+    const schema = Yup.object({
+      dateOfBirth: Yup.string()
+        .notOneOf([INVALID_DATE], 'Invalid date')
+        .label('Date of birth')
+        .required()
+        .test(
+          'min',
+          '${path} must be before ' + formatDate(new Date(min)),
+          (value) => {
+            if (typeof value !== 'string' || value === INVALID_DATE) {
+              return false;
+            }
+
+            const dateOfBirth = new Date(value);
+            const ageInYears = getAgeInYearsFromDate(dateOfBirth);
+
+            if (ageInYears >= 16) {
+              setState('over-sixteen');
+              return true;
+            }
+
+            setState('under-sixteen');
+            return true;
+          }
+        ),
+      title: Yup.string().label('Title').required(),
+      firstName: Yup.string().label('First Name').required(),
+      surname: Yup.string().label('surname').required(),
+      nationalInsuranceNumber: Yup.string()
+        .label('National Insurance Number')
+        .required(),
+    });
+
+    switch (state) {
+      case 'under-sixteen':
+        return schema.pick(['title', 'firstName', 'surname', 'dateOfBirth']);
+      case 'over-sixteen':
+        return schema.pick([
+          'title',
+          'firstName',
+          'surname',
+          'dateOfBirth',
+          'nationalInsuranceNumber',
```

Comment:
We should also need the main applicant scenario where phone / email is required?